chat-widget/src/components/prechatsurveypanestateful/PreChatSurveyPaneStateful.tsx

```diff
@@ -70,7 +70,7 @@ export const PreChatSurveyPaneStateful = (props: IPreChatSurveyPaneStatefulParam
                 if (persistedState &&
                     !isUndefinedOrEmpty(persistedState?.domainStates?.liveChatContext) &&
                     persistedState?.appStates?.conversationState === ConversationState.Active &&
-                    !state.appStates.skipChatButtonRendering) {
+                    state.appStates.hideStartChatButton === false) {
```

Comment:
use !

Reply:
this is an intentional change. All props in v2 are optional, and these values could be `{}, null, undefined, <empty>` depending where are they being set.  So `Strict Equality(===)` operator helps to ensure that the critical code inside only executes when the value is `false` and no other case, provides a more stable code.